app/components/elements/forms/nested_component.html.erb

```diff
@@ -2,6 +2,11 @@
   <legend class="h5">
     <%= header_label %>
   </legend>
+  <% delete_button = capture do %>
```

Comment:
Why DRY this? I initially missed that there were 2 identical copies of the button.

Reply:
I un-DRYed (wet?) it and added a comment to help clarify.